db/migrate/20150429192808_add_scope_to_sequences.rb

```diff
@@ -0,0 +1,5 @@
+class AddScopeToSequences < ActiveRecord::Migration
+  def change
+    add_column :sequences, :scope, :string
```

Comment:
Since we're potentially querying Sequence by the new column, I think ```scope``` should have an index on it.